stdlib/public/core/OverloadMacros.swift

```diff
@@ -0,0 +1,46 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2017 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+#if $Macros && hasAttribute(attached)
+
+
+/*
+
+@_CompatibilityOverload(added: debugName, in: .swift_9999)
```

Comment:
Is this file meant to be in this PR?

Reply:
Whoop, nope not sure how this ended up here, could be some artifact of rebasing "ancient" PR 🤔 

Lemme mark this draft still